crates/symbolicator/src/service.rs

```diff
@@ -28,8 +28,11 @@ use symbolicator_js::interface::{CompletedJsSymbolicationResponse, SymbolicateJs
 use symbolicator_js::SourceMapService;
 use symbolicator_native::interface::{CompletedSymbolicationResponse, SymbolicateStacktraces};
 use symbolicator_native::SymbolicationActor;
+use symbolicator_proguard::interface::{
+    CompletedJvmSymbolicationResponse, SymbolicateJvmStacktraces,
+};
 // TODO: actually use it
-use symbolicator_proguard as _;
+use symbolicator_proguard::{self as _, ProguardService};
```

Comment:
```suggestion
use symbolicator_proguard::{ProguardService};
```

plus remove the comment above :-)

Reply:
Good catch, thanks.